diff --git a/cw_monero/ios/Classes/monero_api.cpp b/cw_monero/ios/Classes/monero_api.cpp
index a62a6646..0c15a42f 100644
--- a/cw_monero/ios/Classes/monero_api.cpp
+++ b/cw_monero/ios/Classes/monero_api.cpp
@@ -311,7 +311,7 @@ extern "C"
 
     bool load_wallet(char *path, char *password, int32_t nettype)
     {
-        nice(19);
+        // nice(19);
         Monero::NetworkType networkType = static_cast<Monero::NetworkType>(nettype);
         Monero::WalletManager *walletManager = Monero::WalletManagerFactory::getWalletManager();
         Monero::Wallet *wallet = walletManager->openWallet(std::string(path), std::string(password), networkType);
@@ -398,7 +398,7 @@ extern "C"
 
     bool connect_to_node(char *error)
     {
-        nice(19);
+        // nice(19);
         bool is_connected = get_current_wallet()->connectToDaemon();
 
         if (!is_connected)
@@ -411,7 +411,7 @@ extern "C"
 
     bool setup_node(char *address, char *login, char *password, bool use_ssl, bool is_light_wallet, char *error)
     {
-        nice(19);
+        // nice(19);
         Monero::Wallet *wallet = get_current_wallet();
         
         std::string _login = "";
@@ -490,7 +490,7 @@ extern "C"
     bool transaction_create(char *address, char *payment_id, char *amount,
                                               uint8_t priority_raw, uint32_t subaddr_account, Utf8Box &error, PendingTransactionRaw &pendingTransaction)
     {
-        nice(19);
+        // nice(19);
         
         auto priority = static_cast<Monero::PendingTransaction::Priority>(priority_raw);
         std::string _payment_id;
@@ -530,7 +530,7 @@ extern "C"
     bool transaction_create_mult_dest(char **addresses, char *payment_id, char **amounts, uint32_t size,
                                                   uint8_t priority_raw, uint32_t subaddr_account, Utf8Box &error, PendingTransactionRaw &pendingTransaction)
     {
-        nice(19);
+        // nice(19);
 
         std::vector<std::string> _addresses;
         std::vector<uint64_t> _amounts;
diff --git a/cw_wownero/ios/Classes/wownero_api.cpp b/cw_wownero/ios/Classes/wownero_api.cpp
index d66316cd..0f4461e2 100644
--- a/cw_wownero/ios/Classes/wownero_api.cpp
+++ b/cw_wownero/ios/Classes/wownero_api.cpp
@@ -410,7 +410,7 @@ extern "C"
 
     bool load_wallet(char *path, char *password, int32_t nettype)
     {
-        nice(19);
+        // nice(19);
         Monero::NetworkType networkType = static_cast<Monero::NetworkType>(nettype);
         Monero::WalletManager *walletManager = Monero::WalletManagerFactory::getWalletManager();
         Monero::Wallet *wallet = walletManager->openWallet(std::string(path), std::string(password), networkType);
@@ -502,7 +502,7 @@ extern "C"
 
     bool connect_to_node(char *error)
     {
-        nice(19);
+        // nice(19);
         bool is_connected = get_current_wallet()->connectToDaemon();
 
         if (!is_connected)
@@ -515,7 +515,7 @@ extern "C"
 
     bool setup_node(char *address, char *login, char *password, bool use_ssl, bool is_light_wallet, char *error)
     {
-        nice(19);
+        // nice(19);
         Monero::Wallet *wallet = get_current_wallet();
 
         std::string _login = "";
@@ -585,7 +585,7 @@ extern "C"
     bool transaction_create(char *address, char *payment_id, char *amount,
                                               uint8_t priority_raw, uint32_t subaddr_account, Utf8Box &error, PendingTransactionRaw &pendingTransaction)
     {
-        nice(19);
+        // nice(19);
 
         auto priority = static_cast<Monero::PendingTransaction::Priority>(priority_raw);
         std::string _payment_id;
@@ -625,7 +625,7 @@ extern "C"
     bool transaction_create_mult_dest(char **addresses, char *payment_id, char **amounts, uint32_t size,
                                                   uint8_t priority_raw, uint32_t subaddr_account, Utf8Box &error, PendingTransactionRaw &pendingTransaction)
     {
-        nice(19);
+        // nice(19);
 
         std::vector<std::string> _addresses;
         std::vector<uint64_t> _amounts;
